frontend/src/editor/automerge_websocket_editor.ts

```diff
@@ -62,17 +62,31 @@ export function useAutomergeWebsocketEditor(
           editor.setDoc(newDoc);
         }
       } else if (msg_type === MessageSyncType.ChangeBacklogComplete) {
+        console.info('backlog complete');
         initialSync = false;
-        console.log(`All changes synced in ${(Date.now() - start) / 1000} s`);
+        const syncCompleteTime = Date.now();
+        console.info(`All changes synced in ${(syncCompleteTime - start) / 1000} s`);
 
         const migratedDoc = migrateDocument(doc as Automerge.Doc<Document>);
         sendDocChange(migratedDoc);
+
         editor.setDoc(migratedDoc);
+        const setDocTime = Date.now();
+        console.info(`setDoc in ${(setDocTime - syncCompleteTime) / 1000} s`);
+
         onInitialSyncComplete();
       } else if (msg_type === MessageSyncType.FullDoc) {
         console.log('Received new document');
         doc = Automerge.load(msg);
-        console.log('Loaded new document');
+        if (!initialSync) {
```

Comment:
Why is this gated on `!initialSync`?

Reply:
because otherwise it will be batch-applied with all the other changes on sync complete